streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableImplTest.java

```diff
@@ -304,6 +363,30 @@ public void testStateStore() {
         }
     }
 
+    @Test
+    public void testSendingOldValuesNotSetIfNotMaterialized() {
```

Comment:
`shouldNotSetSendingOldValuesIfNotMaterialized`

Reply:
I prefer `should` too, but a lot of the code users `test`.  Happy to switch. Can I use `should` everywhere?